src/components/paper.tsx

```diff
@@ -0,0 +1,34 @@
+import {Button, Text, View} from "react-native";
+import {style, theme} from "../util/Style";
+import React from "react";
+
+
+export function Paper({ navigation }: any) {
```

Comment:
das würde ich nicht hardcoden, vielleicht in eine art config auslagen

Reply:
du meinst die Haftungsausschlusserklärung?

Reply:
Ja genau die